doc/data/messages/t/too-many-positional/details.rst

```diff
@@ -1 +0,0 @@
-Reserved message name, not yet implemented.
```

Comment:
Probably need some explanation about how it's hard to remember the order of everything if there's a lot of positional and thus it's a bad API. (We can take some inspiration from what Ruff says)